src/main/java/network/brightspots/rcv/ResultsWriter.java

```diff
@@ -538,33 +541,24 @@ void generateOverallSummaryFiles(
     generateSummaryJson(roundTallies, tallyTransfers, null, outputPath);
   }
 
-  // write CastVoteRecords for the specified contest to the provided folder
+  // write CastVoteRecords for the specified contest to the provided folder,
```

Comment:
So now it's to a single file in the provided folder. In this case, should the user just specify a full file path instead of a folder?

Reply:
This is run automatically on tabulation, where the user specifies a folder for all output files.